share/rocm/cmake/ROCMInstallTargets.cmake

```diff
@@ -157,6 +159,19 @@ function(rocm_install_targets)
                         NAMELINK_ONLY
             )
         endif()
+        if(T_TYPE MATCHES ".*_LIBRARY" AND ROCM_SYMLINK_LIBS AND NOT ROCM_CREATED_${PROJECT_NAME}_SYMLINK
+            AND NOT CMAKE_HOST_WIN32)
+
+            set(ROCM_CREATED_${PROJECT_NAME}_SYMLINK TRUE CACHE INTERNAL "Record symlink created.")
+            string(TOLOWER "${PROJECT_NAME}" LINK_SUBDIR)
```

Comment:
This does assume that the project name and subdir name are the same. I believe this is so on most projects, but is that the case for all projects we have?

Reply:
For all of the math libraries, this is the case, I can't speak to any others.